public/components/security/policies/create-policy.tsx

```diff
@@ -406,37 +406,6 @@ export const CreatePolicyFlyout = ({ closeFlyout }) => {
             {!!addedResources.length && (
               <>
                 <EuiSpacer size="s"></EuiSpacer>
-                <EuiFlexGroup>
-                  <EuiFlexItem>
-                    <EuiInMemoryTable items={addedResources} columns={resources_columns} />
-                  </EuiFlexItem>
-                </EuiFlexGroup>
-              </>
-            )}
-            <EuiSpacer></EuiSpacer>
-            <EuiFormRow label="Select an effect" helpText="Select an effect.">
-              <EuiSuperSelect
-                options={effectOptions}
-                valueOfSelected={effectValue}
-                onChange={(value) => onEffectValueChange(value)}
-              />
-            </EuiFormRow>
-            <EuiSpacer />
-            <EuiButton
-              disabled={
-                !policyName || !addedActions.length || !addedResources.length || !effectValue
-              }
-              onClick={() => {
-                createPolicy();
-              }}
-              fill
-            >
-              Create policy
-            </EuiButton>
-          </EuiForm>
-        </EuiFlyoutBody>
-      </WzFlyout>
-      {modal}
-    </>
+                <EuiFEuiFlyout
```

Comment:
I think the component would be only `EuiFlyout` and should be closed.

Reply:
may be missing the `{modal}`

Reply:
Thank you @yenienserrano. Resolved